toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapViewpointOperation.kt

```diff
@@ -63,13 +66,13 @@ public sealed class MapViewpointOperation {
      * navigation.
      *
      * @property viewpoint the new viewpoint
-     * @property durationSeconds the duration of the animation in seconds
+     * @property durationSeconds the duration of the animation
      * @property curve the animation curve to apply
      * @since 200.4.0
      */
     public class Animate(
         public val viewpoint: Viewpoint,
-        public val durationSeconds: Float = 0.25f,
+        public val durationSeconds: Duration = 0.25.seconds,
```

Comment:
This should just be `duration` now. Users can specify a duration in whatever unit they like. Internally we will convert it to seconds:
```suggestion
        public val duration: Duration = 0.25.seconds,
```

Reply:
I think the setviewpoint micro app will also need updating.